lib/Runtime/Library/BuiltIn/IndexOf.js

```diff
@@ -0,0 +1,48 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+
+"use strict";
+
+(function arrayIndexOf(searchElement, fromIndex) {
+
+    var k;
+
+    if (this == null) {
+        throw new TypeError('"this" is null or not defined');
+    }
+
+    var o = this;
+
+    var len = o.length >>> 0;
+
+    if (o.length <= 0) {
+        return -1;
+    }
+
+    var n = fromIndex | 0;
+
+    if (n >= len) {
+        return -1;
+    }
+
+    if (n >= 0) {
```

Comment:
>n >= 0 [](start = 8, length = 6)

Note -0 and +0 are both === 0.
The spec notes that when n is -0 k should be set to +0. If you do (fromIndex >>> 0) above, -0 will be coerced to +0, so you don't need to do anything here.